src/simtools/schemas/model_parameters/dsum_clipping.schema.yml

```diff
@@ -24,7 +24,6 @@ instrument:
   type:
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for FlashCams?

BTW, note that if we take Konrad's good suggestion to change to MST-FlashCam and MST-NectarCam instead of MSTS and MSTN designs, we will need to change the schemas.

Reply:
I suggest to keep both MSTS/MSTN. There is uncertainty in what is going where.